style.css

```diff
@@ -369,6 +369,185 @@ button:active {
   background: var(--colorButtonPressed);
 }
 
+/* ----- POPUP CREATION ----- */
+
+.popup__window {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--backgroundColor);
+  background-blend-mode: multiply;
+  z-index: 100;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.popup_content {
+  padding: 1.6rem;
+  width: 90%;
+  max-width: 920px;
+  max-height: 1083px;
+  margin: 5% auto 2% auto;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 8px;
+  z-index: 200;
+}
```

Comment:
Great job @mateoburvi 🤝

- [ ] As requested by the previous reviewer [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249248370), for the project button, kindly make sure to place them according to the given design guidelines as suggested  to get a design close to the Figma template as much as possible 💪📌